BUILD.bazel

```diff
@@ -54,10 +54,10 @@ nogo(
     # that are always correct).
     # You can see the what `go vet` does by running `go doc cmd/vet`.
     deps = [
-        "//vendor/github.com/nunnatsa/ginkgolinter:go_default_library",
+        #"//vendor/github.com/nunnatsa/ginkgolinter:go_default_library",
```

Comment:
We should keep track of disabling the nogo plugins, and reenable them at a later point in time.